app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/javalin/DomainsRegistrationTest.groovy

```diff
@@ -97,14 +103,166 @@ class DomainsRegistrationTest extends Specification {
         javalinContext.method() >> HandlerType.POST
 
         when:
-        def javalinHandler = DomainsRegistration.createHandler(rawHandler)
+        def javalinHandler = DomainsRegistration.createHandler(rawHandler, false)
         javalinHandler.handle(javalinContext)
 
         then:
         handlerCalled == true
         1 * javalinContext.status(_ as Integer)
     }
 
+    def "authenticateRequest happy path works"() {
+        given:
+        def expectedDomainResponse = null
+
+        def mockValidator = Mock(AuthRequestValidator)
+        mockValidator.isValidAuthenticatedRequest(_ as DomainRequest) >> true
+        TestApplicationContext.register(AuthRequestValidator, mockValidator)
+        TestApplicationContext.injectRegisteredImplementations()
+
+        when:
+        def actualDomainResponse = DomainsRegistration.authenticateRequest(new DomainRequest())
+
+        then:
+        actualDomainResponse == expectedDomainResponse
+    }
+
+    def "authenticateRequest unauthorized request unhappy path works"() {
+        given:
+        def expectedStatusCode = 401
+
+        def mockValidator = Mock(AuthRequestValidator)
+        mockValidator.isValidAuthenticatedRequest(_ as DomainRequest) >> false
+        TestApplicationContext.register(AuthRequestValidator, mockValidator)
+
+        TestApplicationContext.register(Formatter, Jackson.getInstance())
+
+        TestApplicationContext.injectRegisteredImplementations()
+
+        when:
+        def res = DomainsRegistration.authenticateRequest(new DomainRequest())
+        def actualStatusCode = res.getStatusCode()
+
+        then:
+        noExceptionThrown()
```

Comment:
This line can be removed since you have other checks.  Be default, Spock checks that no exception was thrown.  The only time you need `noExceptionThrown()` is if it is the only check needed because Spock requires at least one check in the `then:` block.